solutions/community-deploy/cloud/AWS/README.md

```diff
@@ -82,15 +91,17 @@ passwd root
 ### CORTX Build
 
 - We will use [cortx-build](https://github.com/Seagate/cortx/pkgs/container/cortx-build) docker image to compile entire CORTX stack.  
-- Login into AWS instance over SSH using IP address from Terraform script execution output
+- Login into AWS instances over SSH using public IP address from terraform script execution output
```

Comment:
We need to perform CORTX build generation steps on only one node.

Reply:
fixed